flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/JsonExistsFunctionITCase.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.planner.functions;
+
+import org.apache.flink.table.api.DataTypes;
+import org.apache.flink.table.expressions.JsonExistsOnError;
+import org.apache.flink.table.functions.BuiltInFunctionDefinitions;
+
+import org.apache.commons.io.IOUtils;
+import org.junit.runners.Parameterized;
+
+import java.io.InputStream;
+import java.nio.charset.Charset;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.apache.flink.table.api.Expressions.$;
+import static org.apache.flink.table.api.Expressions.nullOf;
+
+/** Tests for {@link BuiltInFunctionDefinitions#JSON_EXISTS}. */
+public class JsonExistsFunctionITCase extends BuiltInFunctionTestBase {
+
+    @Parameterized.Parameters(name = "{index}: {0}")
+    public static List<TestSpec> testData() throws Exception {
+        final InputStream jsonResource =
+                JsonExistsFunctionITCase.class.getResourceAsStream("/json/json-exists.json");
+        if (jsonResource == null) {
+            throw new IllegalStateException(
+                    String.format(
+                            "%s: Missing test data.", JsonExistsFunctionITCase.class.getName()));
+        }
+
+        final String jsonValue = IOUtils.toString(jsonResource, Charset.defaultCharset());
```

Comment:
do we have our own Flink util somewhere? depending on an external dependency for such little things should be avoided

Reply:
We do, but it has no methods to just read the entire file content without much hassle (I assume it's more targeted at high-performing implementations). Apache Commons doesn't strike me as a problematic dependency for tests, and is already used for that in various places.